tests/requirements.txt

```diff
@@ -2,3 +2,4 @@
 django-app-helper
 tox
 coverage
+cryptography==3.3.2
```

Comment:
can you pin this < 3.4?
also, why only this one?

Reply:
it apparently only affects Django 3.1+